amundsen_application/models/user.py

```diff
@@ -28,16 +28,11 @@ def load_user(user_data: Dict) -> User:
         # in the user metadata.
         if _str_no_value(user_data.get('profile_url')) and app.config['GET_PROFILE_URL']:
             user_data['profile_url'] = app.config['GET_PROFILE_URL'](user_data['user_id'])
-        data, errors = schema.load(user_data)
-        return data
+        return schema.load(user_data)
     except ValidationError as err:
         return err.messages
 
 
 def dump_user(user: User) -> Dict:
     schema = UserSchema()
-    try:
-        data, errors = schema.dump(user)
-        return data
-    except ValidationError as err:
-        return err.messages
+    return schema.dump(user)
```

Comment:
dump would have never raised `ValidationError` previously. It would have returned errors. So the try/catch was not doing anything.